src/public/map.js

```diff
@@ -41,12 +42,18 @@ function start() {
 
   // Load all MarkLogic feature and industry options for dropdown menus
   // and Draw all map markers
-  doPost('/search.sjs', "", drawPage, true);
+  doPost('/search.sjs', drawPage, true);
 
   // mouse-click event for 'clear map' button
   $("#clearButton").click(removeAllFeatures);
 
-  loadMLInfo();
+  //Selections will hold info on the current state of selected options to query
+  selections = {
+    features: [],
+    industries: [],
+    date1: "",
```

Comment:
startDate and endDate would be much more self-describing than date1 and date2 (unless you're going for a Cat In The Hat reference).